.github/workflows/helm-chart.yml

```diff
@@ -20,3 +20,36 @@ jobs:
         run: nix-shell --run ".ci/helm-docs.sh"
       - name: HelmChart Lint
         run: nix-shell --run ".ci/helm-lint.sh"
+
+  publish:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    needs: ["chart-test"]
+    if: ${{ github.event_name != 'pull_request' }}
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+          fetch-depth: 0
+    - uses: DeterminateSystems/nix-installer-action@v14
+    - uses: DeterminateSystems/magic-nix-cache-action@v8
```

Comment:
I think we can remove the cache, it's not doing anything at the moment and if triggers warnings in the files

Reply:
Yeah, let's remove that in another PR :)

Reply:
I've removed it from other workflows now, so please remove it from this new workflow